sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultPlanCoordinator.java

```diff
@@ -99,7 +99,14 @@ public DefaultPlanCoordinator(
 
     @Override
     public boolean hasOperations() {
-        return planManagers.stream().anyMatch(manager -> !manager.getPlan().isComplete());
+        boolean ret = false;
+        for (final PlanManager planManager : planManagers) {
```

Comment:
Yea, I found it quite useful during debugging.  Not that there's any way to enable DEBUG logging currently...